op-node/flags/p2p_flags.go

```diff
@@ -55,6 +55,30 @@ var (
 	SyncReqRespName        = "p2p.sync.req-resp"
 )
 
+func DeprecatedP2PFlags(envPrefix string) []cli.Flag {
```

Comment:
Why does this func need to be public?

Reply:
oh, p2pdeprecated doesn't. Deprecated does